### DISCUSSION TITLE
Floodfill behind shield or emblem but not text

### DISCUSSION THREAD

**Discussion by ojwebb** [ImageMagick/ImageMagick#2917] [Digital Image Processing]
Hi guys, wondering if anyone has an idea of how I could set up a workflow that would involve floodfilling  background of a logo that had both an emblem mixed with text. Hoping to set parameters like knock out letter holes in objects with one color like text, and leave a white underlay for larger seperate objects with multiple nested colors inside them. Thanks for any help, I appreciate it. 
![7AC98ADC-32EC-465A-9CB2-E759C7581638](https://user-images.githubusercontent.com/62193597/100950229-850d9080-34d1-11eb-9e4b-cb9d6e83d002.png)

**Comment by fmw42** [comment]
I do not understand your question. Please give an example. Also please identify your IM version and platform.

**Reply by ojwebb** [reply]
Sorry I’m running version 7 on the command line, windows. I found another example. Here’s a file that will need both floodfill under character and normal background transparency under text. To get rid of the text holes.
![10E18517-55E1-4D1C-B6F0-B4730FACCF11](https://user-images.githubusercontent.com/62193597/100954822-f6057600-34da-11eb-9e4f-cb7da86890c9.png)

**Comment by fmw42** [comment]
Is this what you want?

`convert shield.png -fuzz 35% -fill red -opaque white shield_red.png
`


![shield_red](https://user-images.githubusercontent.com/13458218/100954696-e893c000-34c9-11eb-8b5e-b4b6b03c8393.png)

See 

https://imagemagick.org/Usage/color_basics/#replace

but in particular

https://imagemagick.org/Usage/color_basics/#floodfill
